torch/csrc/distributed/c10d/quantization/quantization_gpu.cu

```diff
@@ -0,0 +1,147 @@
+#include <caffe2/operators/segment_reduction_op_gpu.cuh>
```

Comment:
what make us using this include? I don't see any call site calling this file

Reply:
Some of the functions and parameters are enabled by this (C10_CUDA_KERNEL_LAUNCH_CHECK, getCurrentCUDAStream, OptionalCUDAGuard, ...). I'm trying to find their sources and include them directly.

Reply:
All these macro/functions are not related to that op, so we shouldn't include that op and instead we should directly include those relevant headers (which I think it should resides somewhere in the torch/c10 directly or torch/cuda directory